### ISSUE TITLE
2.3.1: puppet-lint does not show errors/warnings --error-level option.

### ISSUE THREAD

**Issue by zekefast** [rodjek/puppet-lint#756]
# Environment
- `puppet-lint`: `2.3.1`
- Shell: 
  ```
  $ LANG=C bash --version
  GNU bash, version 4.4.12(1)-release (x86_64-pc-linux-gnu)
  Copyright (C) 2016 Free Software Foundation, Inc.
  License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

  This is free software; you are free to change and redistribute it.
  There is NO WARRANTY, to the extent permitted by law.
  ```
- Operation System: Debian GNU/Linux 9.1 (stretch)
- Kernel: `4.9.0-3-amd64 #1 SMP Debian 4.9.30-2+deb9u5 (2017-09-19) x86_64 GNU/Linux`

# Actual behaviour

```
$ bundle exec puppet-lint --error-level all manifests/nginx.pp
```

No output at all, but set status to `1` when launched with `--fail-on-warnings` option.

# Expected behaviour

It should display errors and warnings in file.

For example, like follows:
```
$ bundle exec puppet-lint --error-level all manifests/nginx.pp
WARNING: indentation of => is not properly aligned (expected in column 13, but found it in column 12) on line 19
```

# Steps to reproduce

Run `puppet-lint` on puppet file with misaligned class properties.
```puppet
file "/tmp/example" {
  ensure => 'present'
  force => true
}
```

Correct version of this file will be
```puppet
file "/tmp/example" {
  ensure => 'present'
  force  => true
}
```

**Comment by patrickconant** [comment]
I ran into this too after updating to 2.3.1 on MacOS High Sierra. I can't tell whether there are actually lint issues in our puppet manifests, or if `--fail-on-warnings` is just causing a return code of `1` regardless of whether there are issues or not. My guess is that there are issues (since I can get a return code of `0` if I alter the input file list), but with no text output from the puppet-lint command I can't tell what the issues might be.

**Comment by cyberious** [comment]
Just ran into this as well, all my CI systems are failing as a result.

**Comment by farkasmate** [comment]
It breaks the Puppet Development Kit as `puppetlabs_spec_helper` installs the latest version.

All the printing seems to be broken.
```
# puppet-lint --version
puppet-lint: no file specified
puppet-lint: try 'puppet-lint --help' for more information
```

**Comment by rnelson0** [comment]
This appears related to #754 which should be addressed in 2.3.2 (#755 and #758), can you confirm?

**Comment by farkasmate** [comment]
It works, thank you.

```
# puppet-lint --version
puppet-lint 2.3.2
```

Also printing all the issues.

**Comment by patrickconant** [comment]
Confirmed, works on MacOS too. Thank you.

**Comment by rnelson0** [comment]
Thanks for confirmation!

**Comment by zekefast** [comment]
@farkasmate I also noticed that problem with `puppet-lint --version`, but `2.3.2` fixed it.

On `2.3.1`:
```
$ puppet-lint --version
puppet-lint: no file specified
puppet-lint: try 'puppet-lint --help' for more information
```

On `2.3.2`:
```
$ puppet-lint --version
puppet-lint 2.3.2
```
